src/script/main/app.ts

```diff
@@ -695,6 +696,13 @@ class App {
       window.location.replace(redirect);
     }
 
+    const conversationRedirect = localStorage.getItem(App.LOCAL_STORAGE_LOGIN_CONVERSATION_KEY);
+    if (conversationRedirect) {
+      const {conversation, domain} = JSON.parse(conversationRedirect)?.data;
+      localStorage.removeItem(App.LOCAL_STORAGE_LOGIN_CONVERSATION_KEY);
+      window.location.replace(`#/conversation/${conversation}${domain && `/${domain}`}`);
```

Comment:
In this case ${domain && /${domain}}, if domain === undefined you get on the end of the link #/conversation/<conversation>undefined, please change to ${domain ? /${domain} : ''} to avoid undefined on the end :)